core/src/database/common.rs

```diff
@@ -130,6 +130,26 @@ impl Database {
         Ok(())
     }
 
+    pub async fn save_timeout_tx_sigs(
```

Comment:
Plus encoding and decoding into bytea type caused headaches too many times.

Reply:
Fixed in https://github.com/chainwayxyz/clementine/pull/361/commits/090216c71cacc62cbc0b11af480cb680614ee100. Also signatures are serialized inside the database crate because it's database crate's job.